starter/src/templates/dev.tsx

```diff
@@ -0,0 +1,131 @@
+import "@yext/visual-editor/style.css";
+import {
+  Template,
+  TemplateConfig,
+  GetPath,
+  TemplateProps,
+  TemplateRenderProps,
+  GetHeadConfig,
+  HeadConfig,
+} from "@yext/pages";
+import { componentRegistry } from "../ve.config";
+import {
+  applyTheme,
+  Editor,
+  VisualEditorProvider,
+  YextSchemaField,
+} from "@yext/visual-editor";
+import { themeConfig } from "../../theme.config";
+import { buildSchema } from "../utils/buildSchema.ts";
+import tailwindConfig from "../../tailwind.config";
+import { devTemplateStream } from "../dev.config";
+
+export const config = {
+  name: "dev-location",
+  stream: {
+    $id: "dev-location-stream",
+    filter: {
+      entityTypes: ["location"],
+    },
+    fields: [
+      "id",
+      "uid",
+      "meta",
+      "slug",
+      "name",
+      "hours",
+      "dineInHours",
+      "driveThroughHours",
+      "address",
+      "yextDisplayCoordinate",
+      "c_productSection.sectionTitle",
+      "c_productSection.linkedProducts.name",
+      "c_productSection.linkedProducts.c_productPromo",
+      "c_productSection.linkedProducts.c_description",
+      "c_productSection.linkedProducts.c_coverPhoto",
+      "c_productSection.linkedProducts.c_productCTA",
+      "c_hero",
+      "c_faqSection.linkedFAQs.question",
+      "c_faqSection.linkedFAQs.answerV2",
+      "additionalHoursText",
+      "mainPhone",
+      "emails",
+      "services",
+      "c_deliveryPromo",
+    ],
+    localization: {
+      locales: ["en"],
+    },
+  },
+  additionalProperties: {
+    isVETemplate: true,
+  },
+} as const satisfies TemplateConfig;
+export const transformProps = async (data: any) => {
+  const emptyLayout = {
+    root: {},
+    content: [],
+    zones: {},
+  };
+  const updatedDocument = data;
+  if (!updatedDocument.document.__) {
```

Comment:
Is this always going to be true? The actual documents we use should be checked in so none of this should be necessary. Ensure that the npm run dev cmd uses the local and no-prod-url flags.